cdk/visit/__init__.py

```diff
@@ -113,13 +116,14 @@ def log_visit_lambda(self, table_name: str):
             code=aws_lambda.Code.from_asset('visit/lambda_code'),
             environment={
                 'TABLE_NAME': table_name,
+                'API_ENDPOINT': api_endpoint,
```

Comment:
It looks like we already have this information stored in member variables and they might not need to be passed in via function parameters. We use `self.zones.visit.zone_name` in our cloudfront function on line 77. I would consider doing the same thing here

Reply:
Yeah I see what you mean, I went ahead and cleaned it up a bit.